.github/workflows/create-test-account.yml

```
@@ -0,0 +1,36 @@
+name: Create Test Account
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_cli:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+
+    - run: make dist-ci
+    - run: make test
+
+  test_action:
```

How about a different name here?

```suggestion
  create_test_account:
```